.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,47 @@
+name: Bug report
+description: "Create a report to help us improve"
+title: "[BUG]: "
+labels: ["bug"]
+body:
+   - type: markdown
+    attributes:
+      value: "### Before you open an issue, please verify if a similar one already exists or has been closed before. More details about the process of contributing can be found in [CONTRIBUTING.md](https://github.com/TheAlgorithms/JavaScript/blob/master/CONTRIBUTING.md)."
+   - type: textarea
```

I believe we need to set an ID name for each of the `textarea` fields.
```suggestion
   - type: textarea
    id: description
```